Open Judge System/Web/OJS.Web/Web.config

```diff
@@ -289,7 +291,15 @@
     </staticContent>
     <directoryBrowse enabled="false" />
     <rewrite>
-      <rules />
+      <rules>
+        <rule name="Redirect to HTTPS" stopProcessing="true">
+          <match url="(.*)" />
+          <conditions>
+            <add input="{HTTPS}" pattern="off" />
+          </conditions>
+          <action type="Redirect" url="https://{HTTP_HOST}/{R:1}" redirectType="Permanent" />
+        </rule>
+      </rules>
```

Comment:
We have this configured in Web.Release.config. See https://github.com/SoftUni-Internal/suls/blob/staging/Documentation/JudgeWeb-Web.Release.config.
Is there any problems with this for local development? It could stay if it's ok.

Reply:
I didn't know that there is a place where we store the release config. I noticed it on PROD but I added it in the solution to be sure that we will not forget to add it in the config in order to know what we need when rollout a new instance of Judge. 
I will remove it from this config.